src/queues/sqs/mod.rs

```diff
@@ -97,9 +97,13 @@ impl Factory for Queue {
         let client: Box<Sqs> = if let Some(ref keys) = settings.aws.keys {
             let creds =
                 StaticProvider::new(keys.access.0.clone(), keys.secret.0.clone(), None, None);
-            Box::new(SqsClient::new(RequestDispatcher::default(), creds, region))
+            Box::new(SqsClient::new_with(
+                HttpClient::new().expect("Couldn't start HTTP Client."),
```

Comment:
I'm not really sure about this `expect` here, what do you think @philbooth ? Should I change this functions to return a `Result` and handle the errors or panicking is alright in this case?

Reply:
I think panicking in this case is fine because it's initialisation code, so the process will abort on startup rather than later when we think everything is working correctly.